README.md

```diff
@@ -2,7 +2,7 @@
 
 A Ruby library to expand shortened URLs.
 
-**Current version:** 1.2.3  
+**Current version:** 1.2.4
```

Comment:
Can you restore the two trailing spaces needed here to insert a [Markdown line-break](https://daringfireball.net/projects/markdown/syntax#p)?